### PR TITLE
feat: Implement in call reactions (WPB-10158)

### PR DESCRIPTION
**Pull request by thisisamir98**
## Description
This PR implements the in-call reactions feature, enhancing user interactions during calls by allowing emoji reactions.

https://github.com/wireapp/generic-message-proto/pull/88

## Screenshots/Screencast (for UI changes)

https://github.com/user-attachments/assets/06326468-c03a-4761-ba5e-e90fdf78ff83

### CODE REVIEW

File: src/script/calling/CallingRepository.ts [hunk 1]
```diff
@@ -723,6 +724,42 @@ export class CallingRepository {
...
+        });
+
+        this.callState.emojis([...this.callState.emojis(), ...newEmojis]);
+
+        setTimeout(() => {
+          const remainingEmojis = this.callState
+            .emojis()
+            .filter(item => !newEmojis.some(newItem => newItem.id === item.id));
+          this.callState.emojis(remainingEmojis);
+        }, 4000);
```
Comment (przemvs) [inline src/script/calling/CallingRepository.ts]: Please move time to static variable :)

File: src/script/components/calling/FullscreenVideoCall.tsx [hunk 2]
```diff
@@ -272,6 +282,18 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
...
+    setDisabledEmojis(prev => [...prev, selectedEmoji]);
+
+    sendEmoji(selectedEmoji, call);
+
+    setTimeout(() => {
+      setDisabledEmojis(prev => [...prev].filter(emoji => emoji !== selectedEmoji));
+    }, 4000);
+  };
+
+  console.info('bardia', {disabledEmojis});
```
Comment (przemvs) [inline src/script/components/calling/FullscreenVideoCall.tsx]: Probably You forgot to remove it ;D

File: src/script/components/calling/FullscreenVideoCall.tsx [hunk 3]
```diff
@@ -272,6 +282,18 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
...
   };
 
+  const onEmojiClick = (selectedEmoji: string) => {
+    setDisabledEmojis(prev => [...prev, selectedEmoji]);
+
+    sendEmoji(selectedEmoji, call);
+
+    setTimeout(() => {
+      setDisabledEmojis(prev => [...prev].filter(emoji => emoji !== selectedEmoji));
+    }, 4000);
```
Comment (przemvs) [inline src/script/components/calling/FullscreenVideoCall.tsx]: Use static variable for timeout

File: src/script/components/calling/FullscreenVideoCall.tsx (hunk 2 above)
Comment (thisisamir98) [reply src/script/components/calling/FullscreenVideoCall.tsx]: https://github.com/wireapp/wire-webapp/pull/17764/commits/395ae081c4bea65be334d8ec514312aeef6f6c22

File: src/script/components/calling/FullscreenVideoCall.tsx (hunk 3 above)
Comment (thisisamir98) [reply src/script/components/calling/FullscreenVideoCall.tsx]: https://github.com/wireapp/wire-webapp/pull/17764/commits/d00497c32792284aae2dbf180de8d5e1d38dca8d

File: src/script/calling/CallingRepository.ts (hunk 1 above)
Comment (thisisamir98) [reply src/script/calling/CallingRepository.ts]: https://github.com/wireapp/wire-webapp/pull/17764/commits/d00497c32792284aae2dbf180de8d5e1d38dca8d

**Review by przemvs — APPROVED** [review]

**Review by tlebon — APPROVED** [review]
seems fine to me now 👨🏻‍💻👍🏻

**Pull request merged**